plaso/engine/artifact_filters.py

```diff
@@ -20,23 +20,29 @@ class ArtifactDefinitionsFilterHelper(object):
   https://github.com/ForensicArtifacts/artifacts/blob/master/docs/Artifacts%20definition%20format%20and%20style%20guide.asciidoc
   """
 
-  _KNOWLEDGE_BASE_VALUE = 'ARTIFACT_FILTERS'
+  KNOWLEDGE_BASE_VALUE = 'ARTIFACT_FILTERS'
 
-  _COMPATIBLE_REGISTRY_KEY_PATH_PREFIXES = ['HKEY_LOCAL_MACHINE']
+  COMPATIBLE_REGISTRY_KEY_PATH_PREFIXES = frozenset([
```

Comment:
Why change these? Are these intended to be used outside the class (except by unit tests)?

Reply:
I use them in winreg.py as well, but just refactored this a bit into a function so reverting this.